x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/AuthenticatorChain.java

```diff
@@ -176,18 +170,7 @@ private static BiConsumer<Authenticator, ActionListener<AuthenticationResult<Aut
                     }
                 }
                 listener.onFailure(e);
-            };
```

Comment:
Just a reordering to remove the exception consumer variable (and have it as a lambda to `ActionListener#wrap`, which is the more common ordering).